zk-kyc/README.md

```diff
@@ -0,0 +1,85 @@
+# zk-KYC Demonstration Application
+
+This demo leverages ID.me api to generate a client authentication token. The token includes a nonce that is associated with the user's connected wallet address, employing principles from the [OpenPubkey: Augmenting OpenID Connect with User held Signing Keys](https://eprint.iacr.org/2023/296) paper. The JWT's integrity is then verified within the guest using ID.me's public RS256 signing [certificates](https://api.id.me/oidc/.well-known/jwks). Subsequently, the guest generates a cryptographic proof of the JWT's integrity and issues a receipt. This receipt, containing the SNARK, an obfuscated identifier, and the user's address, can be validated on the onchain for ERC721 token minting or other transactions, if valid.
+
+For detailed information about the RISC Zero zkVM, refer to our developer [documentation](https://dev.risczero.com/api).
+
+### Getting Started
+
+#### Project Structure
+
+The ERC721 Token Minting Demo consists of three key components, forming the complete end-user application:
+
+- [`zkvm`]: This directory contains the host, guest, and OIDC library components.
+    - [`zkvm/host`]: The entry point for the web socket client using the [bonsai-sdk] crate. 
+    - [`zkvm/methods`]: The guest source code that runs and is proven in the zkVM.
+    - [`zkvm/oidc-validator`]: A library for implementing JWT validation in the guest and host using the [jwt-compact] crate.
+- [`contracts`]: This directory houses the [foundry] project for smart contracts, scripts, and tests related to ERC721 token minting.
+- [`app`]: The front-end react application directory, enabling end-user interaction.
+
+[`zkvm`]: zkvm/
+[`zkvm/host`]: zkvm/host
+[`zkvm/methods`]: zkvm/methods
+[`zkvm/oidc-validator`]: zkvm/oidc-validator
+[`contracts`]: contracts/
+[`app`]: app/
+[bonsai-sdk]: https://crates.io/crates/bonsai-sdk
+[jwt-compact]: https://github.com/slowli/jwt-compact
+[foundry]: https://github.com/foundry-rs/foundry
+
+#### Application Setup
+
```

Comment:
Any installation pre-requisites? Do I need to install the RISC Zero toolchain?

Reply:
You do not need the toolchain, as it relies on Bonsai